### PR TITLE
ReadMe v1.0

### PR DESCRIPTION
**Pull request by CoCodrilo1710**
First version of ReadMe.md for our Project.

### CODE REVIEW

**Review by DragosGhinea — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -37,12 +37,12 @@ Acest proiect își propune să creeze un joc de apărare cu turnuri în 2D, cu
 - Inamicii devin din ce în ce mai numeroși și mai puternici pe parcursul jocului.
 - Inamicii au abilitatea de a:
   - Distrugere orice tip de clădire în oraș.
```
Comment (DafinaTrufas) [inline README.md]: "distruge" in loc de "distrugere"

File: README.md [hunk 2]
```diff
@@ -37,12 +37,12 @@ Acest proiect își propune să creeze un joc de apărare cu turnuri în 2D, cu
...
-  - Alege singuri traseul, schimbându-l dacă este blocat.
-  - Se adaptează în timp la dificultățile întâlnite.
+  - Alege singuri traseul, schimbându-l dacă este blocat (în funcție de inteligența inamicului).
+  - Se adaptează în timp la dificultățile întâlnite (ex. capătă rezistență la anumite turnuri).
```
Comment (DafinaTrufas) [inline README.md]: "se adapta" in loc de "se adapteaza"

File: README.md [hunk 3]
```diff
@@ -37,12 +37,12 @@ Acest proiect își propune să creeze un joc de apărare cu turnuri în 2D, cu
...
 
-## Funcționalități Viitoare
+## Funcționalități în Dezvoltare
 
-Există funcționalități suplimentare care sunt încă în dezvoltare și încă nu sunt complet definite în povestea jocului:
+Există funcționalități care se vor defini pe măsură ce se lucrează la ele și încă nu sunt complet proiectate în povestea jocului:
```
Comment (DafinaTrufas) [inline README.md]: As zice "definite" in loc de "proiectate" aici.

Comment (DafinaTrufas) [reply README.md]: si "descrierea" in loc de "povestea"

**Pull request merged**